### PR TITLE
Display profile

### LINKED ISSUE #64: Display Profile Media Files
Display profile media files

### PR DESCRIPTION
**Pull request by angel23v**
## Quick Info
Every time you click on each button you will see content filtered by the button. It could be posts, images or videos

## Screenshots
Posts
Screenshot from 2024-04-16 18-51-51
Images
Screenshot from 2024-04-16 19-00-10

### CODE REVIEW

**Review by Zama15 — APPROVED** [review]
LGTM

**Review by PudinDeveloper — APPROVED** [review]

File: app/javascript/controllers/profile_controller.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      e.preventDefault();
+      posts.classList.add("hidden");
+      images.classList.remove("hidden");
+    });
+
+    post.addEventListener("click", (e) => {
+      posts.classList.remove("hidden");
+      images.classList.add("hidden");
+    });
+  }
```
Comment (MauricioNC) [inline app/javascript/controllers/profile_controller.js]: You don't have an `e.preventDefault();` in the last event, is this correct?

**Review by MauricioNC — APPROVED** [review]
LGTM!

File: app/javascript/controllers/profile_controller.js (hunk 1 above)
Comment (angel23v) [reply app/javascript/controllers/profile_controller.js]: Thats's right let me fix it

**Pull request merged**